linux-build/makefile

```diff
@@ -0,0 +1,27 @@
+COMMIT_SHA_SHORT ?= $(shell git rev-parse --short=12 HEAD)
+PWD_DIR:= ${CURDIR}
+SHELL := /bin/bash
+
+default: help;
+
+# ======================================================================================
+build-builder: ## build the builder image that contains the source code
+	@docker build -f build-linux.dockerfile -t mcmap-builder:latest ./..
+
+build-linux: build-builder ## build for linux using docker
```

Comment:
This steps could also be out into a build script instead of having them in the make file.
furthermore build-linux and package-linux could probably unified into a single target.
similar to what i did here: https://github.com/andresbott/minecraft-overviewer-package/blob/main/targets/bullseye-64/build.sh